src/clandinin_lab_to_nwb/brezovec/brezovec_metadata.yaml

```diff
@@ -9,8 +9,9 @@ NWBFile:
     - Brezovec, Luke
     - Berger, Andrew
     - Druckmann, Saul
-    - Clandinin, Thomas 
+    - Clandinin, Thomas
 Subject:
-  species: Drosophila
-  age: TBD  # in ISO 8601, such as "P1W2D"
-  sex: TBD  # One of M, F, U, or O
+  subject_id: TBD  # A unique identifier for the subject
+  species: Drosophila melanogaster
+  age: 3-4D  # in ISO 8601, such as "P1W2D"
```

Comment:
What are you trying to say here? Between 3 and 4 days old? that would be "3D/4D"